### DISCUSSION TITLE
Jaeger Tracing - Same Trace Id but span ids are not stitched together

### DISCUSSION THREAD

**Discussion by Leela-SR** [jaegertracing/jaeger#4071] [Q&A]
Hi Team,

I would like to know when the Trace will have same trace id and why the spans are not stitched together. Please see below screen shot for more details.
![image](https://user-images.githubusercontent.com/112939073/203979034-e06b7231-ad86-4190-b0fb-73676f3bc11a.png)

I expect the image to be like this:
![image](https://user-images.githubusercontent.com/112939073/203979381-51758c14-36c2-4c84-bb79-7b07e6406be8.png)

**Comment by Leela-SR** [comment]
Details on collecting Traces:
We are using OpenTelemetry for Tracing and OtlpCollector for collecting the traces and displaying in Jaeger.

**Comment by yurishkuro** [comment]
This is an issue with your instrumentation (likely not propagating trace context), not with Jaeger.

**Reply by Leela-SR** [reply]
Thanks Yuri, I just fixed the issue.
